ui/pages/onboarding-flow/privacy-settings/privacy-settings.test.js

```diff
@@ -84,19 +109,26 @@ describe('Privacy Settings Onboarding View', () => {
 
     const toggles = container.querySelectorAll('input[type=checkbox]');
     const submitButton = getByText('Done');
+    // TODO: refactor this toggle array, not very readable
     // toggle to false
```

Comment:
Why does this PR change multiple lines in this file, but the PR against develop only changed 1? https://github.com/MetaMask/metamask-extension/pull/24369/files#diff-f116067285f9f2c06bc23f981e6a116a36fd62a30e83a3854aca63e6c8b18811L118